packages/mobile/src/harmony-native/components/input/TextInput/TextInput.tsx

```diff
@@ -237,113 +238,128 @@ export const TextInput = forwardRef(
           pointerEvents={disabled || _disablePointerEvents ? 'none' : undefined}
         >
           <GestureDetector gesture={tap}>
-            <AnimatedFlex
-              h={isSmall ? 34 : 64}
-              w='100%'
-              direction='row'
-              alignItems='center'
-              border={disabled ? 'default' : 'strong'}
-              borderRadius='s'
-              backgroundColor='surface1'
-              ph='l'
-              gap={isSmall ? 's' : 'm'}
-              style={animatedRootStyles}
+            <InsetShadow
```

Comment:
Nice, I assume this works pretty much the same way as web?

Reply:
yeah its not a perfect match, since web is a combination of shadows, but its pretty decent

Reply:
ah yeah that makes sense